tests/serialization/test_parameter_serialization.py

```diff
@@ -1,24 +1,15 @@
 """Test serialization of parameters."""
```

Comment:
Yes, it's the former. In fact, I also thought about this issue already before but decided not to bother too much :D But now that you also bring it up, I guess this is valid point. Have a look at my new version that I'll push in a few mins. I guess you will like it a lot better 👍🏼

Reply:
PS: I would still keep the `parameters` strategy in the strategies file, because I think it will turn out to be useful in other test contexts, where you simply need "some random parameter" ... But it's no longer used for the serialization test in my new version

Reply:
Happy to do so, but this is much easier discussed in a meeting. Can we postpone it till then? I will leave this thread open until we've discussed it ....

Reply:
yes lets discuss in the meeting, for such drastic changes imo that should prob even happen before implementation starts